README.md

```diff
@@ -84,9 +66,17 @@ executable, which is the model code already linked with the ROOT-sim library.
 `rootsim-cc` ultimately relies on `gcc`, so any flag supported by
 `gcc` can be passed to `rootsim-cc`.
 
-To test the correctness of the model, it can be run sequentially, typing
+To test the correctness of the model, it can be run sequentially, typing    
 `./model --sequential --nprc <number of required LPs>`
 This allows to spot errors in the implementation more easily.
 
-Then, to run it in parallel, type
+Then, to run it in parallel, type    
 `./model --np <number of available Cores> --nprc <number of required LPs>`
+
+To run in a distributed environment, you can use standard MPI commands,
+such as:    
+`mpiexec -n 2 -f hosts ./model --np 2 --nprc 16`
```

Comment:
are we sure of `-f` ? The documentation says `--hostfile`

Reply:
ps: that's a good case in which no "oversubscription" on nodes occurs...maybe it's clearer if we write also `--map-by node`?
But it's not really important.

Reply:
Correct!